addon/-private/service-injections.js

```diff
@@ -0,0 +1,15 @@
+const SERVICE_INjECTIONS = new WeakMap();
```

Comment:
Cannot tell if the lower j is on purpose here...

Reply:
all the cool kids are doing it